botbot_plugins/tests/test_vote.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+import pytest
+from botbot_plugins.base import DummyApp
+from botbot_plugins.plugins import vote
+
+
+@pytest.fixture
+def app():
+    return DummyApp(test_plugin=vote.Plugin(), command_prefix="!")
+
+def test_no_concurrent_voting(app):
+    assert app.respond("!startvote") == ["Voting has started."]
```

Comment:
You should use self.assertEqual here and move this under a class that inherits from unittest.TestCase